tests/integration/ceph_base_object_test.go

```diff
@@ -149,6 +153,17 @@ func checkCephObjectUser(
 		assert.NoError(s.T(), err)
 		assert.Equal(s.T(), k8sutil.ReadyStatus, phase)
 	}
+	if checkQuotaAndCaps {
+		// following fields in CephObjectStoreUser CRD doesn't exist before Rook v1.7
```

Comment:
`v1.7.?` what patch release? Should we be introducing a new feature in the middle of a release like this?

@travisn should we just wait for this in v1.8?

Reply:
I'd say it's ok to backport to 1.7 since it's an additive feature with no breaking changes. It's still a while until the 1.8 release will be out, so I wouldn't want to wait that long if it's not breaking.

Reply:
@BlaineEXE : Added the documentation in `ceph-object-store-crd.md`, anyplace else needed?